Add Changelog for 0.2.0 (#962)
AlexsLemonade/OpenScPCA-analysis: CHANGELOG.md

You may want to add temporary notes here for tracking as features are added, before a new release is ready.
-->

## v0.2.0

This release adds the first set of community-contributed analyses to the repository.
These modules are focused on cell type identification and annotation for specific ScPCA datasets.

> Note that many of these modules are still in development at this stage, and may not be fully functional.

Reviewer: ```suggestion
Note that many of these modules are still in development at this stage and may not be fully functional.
```

> Note that many of these modules are still in development at this stage, and may not yet be fully functional.

Reviewer 2: Maybe remove "yet" since we don't necessarily know it will come to pass? Or, replace with "currently" or similar?
```suggestion
Note that many of these modules are still in development at this stage, and may not be fully functional.
```

- `cell-type-dsrct`

> - `cell-type-ETO-ALL-03`

Reviewer 2: ```suggestion
- `cell-type-ETP-ALL-03`
```

- `cell-type-glioblastoma`
- `cell-type-nonETP-ALL-03`
- `cell-type-wilms-tumor-06`

> - `cell-type-wilms0tumor-14`

Reviewer 2: ```suggestion
- `cell-type-wilms-tumor-14`
```

> This release also adds the following new modules developed by the Data Lab:

Reviewer 2: `cell-type-consensus` is also getting added with this release

- `hello-clusters`: a demonstration module for clustering analysis using the [`rOpenScPCA` package](https://github.com/AlexsLemonade/rOpenScPCA)
- `seurat-conversion`: a module for converting `SingleCellExperiment` objects to Seurat objects, also using the `rOpenScPCA` package

> - `metacells`: a module that begins to explore the utility of metacell analysis within the ScPCA project

Reviewer 2: This did already exist at the last release, but didn't have content yet https://github.com/AlexsLemonade/OpenScPCA-analysis/tree/v0.1.0/analyses/metacells

Author: For some reason, I thought it wasn't in the release notes, but it was...
```suggestion
```

Other updates in this release include:

- a new `sync-results.py` script to simplify uploading (and downloading) analysis results from an analysis module to a user's S3 bucket
- changes from miniconda to miniforge for conda usage throughout the project

While not part of this repository, we do want to also note that we have created the [`rOpenScPCA` package](https://github.com/AlexsLemonade/rOpenScPCA), which will house utility functions commonly used by analysis modules here.

> The goal is to centralize common functions used across analyses to make it easier to share code across modules and to maintain consistency in the analyses.

Reviewer 2: tried to tighten this up a bit
```suggestion
The goal is to centralize common functions used across analysis modules to make it easier to share code and main consistency across modules.
```
